### PR TITLE
Vitaliy tolochniy labs

### PR DESCRIPTION
**Pull request by Vitaliy-Kovalevskiy**
Lab 6

### CODE REVIEW

File: labs/vitaliy_tolochniy/Lab5/3.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    n = int(input("Введите длину строк в своей матрице: "))
+    for i in range(0, num):
+        list1.append(module.RandInt(n))
+'''for i in list1:
+    module.OutputList(i)'''
+
+
+
+'''Поиск и нахождение суммы чисел кратных трём'''
+S = 0
```
Comment (kiselhof) [inline labs/vitaliy_tolochniy/Lab5/3.py:26]: Variables start with a lowercase letter

File: labs/vitaliy_tolochniy/Lab5/1.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+'''Лабораторная работа 5
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/Lab5/1.py:1]: Docstrings is in triple double-quoted strings

"""
foo bar
"""

File: labs/vitaliy_tolochniy/Lab5/1.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+'''))
+
+    if a == 1:
+        listt = module.InputList()
+    elif a == 2:
+        b = int(input('Введите длину списка:'))
+        listt = module.RandInt(b)
+    elif a == 3:
+        if len(listt) == 0:
+            print("Вы еще не создали ни одного списка")
```
Comment (kiselhof) [inline labs/vitaliy_tolochniy/Lab5/1.py:36]: Quotes are preferable to be same styled

File: labs/vitaliy_tolochniy/lab6/1.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+'''Лабораторная работа 6
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab6/1.py]: The same. """, not '''

File: labs/vitaliy_tolochniy/lab6/1.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Толочный Виталий
+Розробіть функції для здійснення наступних операцій зі списками:
+Описати функцію DigitCountSum (K, C, S), що знаходить кількість C цифр цілого
+позитивного числа K, а також їх суму S (K - вхідний, C і S - вихідні параметри
+цілого типу). За допомогою цієї функції знайти кількість і суму цифр для кожного
+з п'яти даних цілих чисел.'''
+
+def DigitCountSum (K):
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab6/1.py]: no space before ()

File: labs/vitaliy_tolochniy/lab6/4.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
+'''Лабораторная работа 6
+Толочный Виталий 141
+Описати функцію Cos1 (x, ε) дійсного типу (параметри x, ε - дійсні, ε> 0),
+знаходить наближене значення функції cos (x):
+cos (x) = 1 - x2/ (2!) + x4/ (4!) - ... + (-1) n· x2 · n / ((2 · n)!) + ....
+В сумі враховувати всі складові, модуль яких більше ε. За допомогою Cos1
+знайти наближене значення косинуса для даного x при шести даних ε. '''
+
+def Cos1(x, e):
+    S = 1
```
Comment (kiselhof) [inline labs/vitaliy_tolochniy/lab6/4.py]: Variable name should be meaningfull

File: labs/vitaliy_tolochniy/lab6/2.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+чисел від A до B включно (A і B - цілі). Якщо A> B, то функція повертає 0. За
+допомогою цієї функції знайти суми чисел від A до B і від B до C, якщо дано
+числа A, B, C. '''
+
+def SumRange (A, B):
+    if A > B:
+        return 0
+    S = 0
+    for i in range(A, B+1):
+        S+=i
```
Comment (sen-den) [inline labs/vitaliy_tolochniy/lab6/2.py]: += operator should be surrounded by spaces

Comment (sen-den) [reply labs/vitaliy_tolochniy/lab6/2.py]: most of the binary operators should

File: labs/vitaliy_tolochniy/lab6/3.py [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+for i in l1:
+    print(round(RadToDeg(i),2))
+
+
+
+
+
+
+
+
```
Comment (kiselhof) [inline labs/vitaliy_tolochniy/lab6/3.py]: Use ctrl+alt+l

Comment (sen-den) [inline labs/vitaliy_tolochniy/lab6/3.py]: wtf?

**Review by kiselhof — DISMISSED** [review]

**Review by sen-den — DISMISSED** [review]
Be sure not to do the same formatting mistakes in the future

**Review by sen-den — APPROVED** [review]

**Review by andxzrey — APPROVED** [review]

**Pull request merged**